core/src/main/resources/hudson/model/UpdateCenter/CompleteBatchJob/row.jelly

```diff
@@ -1,7 +1,6 @@
 <?jelly escape-by-default='true'?>
 <j:jelly xmlns:j="jelly:core" xmlns:st="jelly:stapler" xmlns:d="jelly:define" xmlns:l="/lib/layout" xmlns:t="/lib/hudson" xmlns:f="/lib/form">
     <tr id="row${it.id}">
-        <td style="vertical-align: top; padding-right:1em">${%Loading plugin extensions}</td>
```

Comment:
This change seems wrong.

Because the status row remains, users might still wonder what Jenkins is doing:

> <img width="578" alt="Screenshot 2022-09-01 at 07 38 03" src="https://user-images.githubusercontent.com/1831569/187839610-60674538-0bce-47e0-a70c-65e740a11dce.png">

When actually installing something, this entry looks weird compared to the rest:

> <img width="528" alt="Screenshot 2022-09-01 at 07 41 02" src="https://user-images.githubusercontent.com/1831569/187839894-c2dd6e4e-c430-4457-b8a5-fe94ddfe2acd.png">

Additionally, rows are expected to have two columns, so the next cell should have colspan.

As this step can take a while to finish when installing a lot of plugins at the same time, even removing the entire row would be misleading, as something is still being done without visible status.